SecondHomework/Logistic/ParcelManager.cs

```diff
@@ -1,134 +1,37 @@
+﻿public class ParcelManager
 {
-    private const string _filePath = @"C:\Users\kharc\source\repos\SecondHomework\Logistic\Logistic\Parcels.json";
 
-    private List<Parcel> parcels;
+    private List<Parcel> parcels = new List<Parcel>();
+    private FileManager fileManager = new FileManager();
 
     public ParcelManager()
     {
         parcels = new List<Parcel>();
     }
 
-    public void AddParcelFromInput()
-    {
-        Console.Write("What are you planning to send? ");
-        string? name = Console.ReadLine();
-
-        Console.Write("Who is the recipient? ");
-        string? recipient = Console.ReadLine();
-
-        Console.Write("Enter the destination, please: ");
-        string? destination = Console.ReadLine();
-
-        AddParcel(name, recipient, destination);
-    }
-
-    public void Display()
-    {
-        if (parcels.Count == 0)
-        {
-            Logger.NoParcels();
-        }
-        else
-        {
-            foreach (var parcel in parcels)
-            {
-                Console.WriteLine(parcel);
-            }
-        }
-    }
-
-    public void AddParcel(string? name, string? recipient, string? destination)
+    public void AddParcel(Parcel parcel)
     {
-        var parcel = new Parcel(Guid.NewGuid(), name, recipient, destination);
         parcels.Add(parcel);
-        SaveParcels();
     }
 
     public void SaveParcels()
     {
-        var json = JsonConvert.SerializeObject(parcels, Newtonsoft.Json.Formatting.Indented);
-        File.WriteAllText(_filePath, json);
+        fileManager.Save(parcels);
     }
 
-    public void CheckIfDeleteAndRemove()
+    public List<Parcel> ReadParcels()
     {
-        string? answer = Console.ReadLine();
-        if (answer?.ToLower() == "yes")
-        {
-            Logger.CheckForIdToRemove();
-            string? id = Console.ReadLine();
-            if (Guid.TryParse(id, out Guid Id))
-            {
-                DeleteParcels(Id);
-                Console.WriteLine("Updated parcels are:");
-                Display();
-            }
-            else
-            {
-                Console.WriteLine("Invalid ID format.");
-            }
-        }
-        else if (answer?.ToLower() == "no")
-        {
-            Console.WriteLine("Have a nice day!");
-        }
-        else if (string.IsNullOrWhiteSpace(answer))
-        {
-            Console.WriteLine("Don't be silent, i've got a lot of work");
-        }
+        parcels = fileManager.Read();
+        return parcels;
     }
 
-    public void DeleteParcels(Guid id)
+    public void RemoveParcels(Guid id)
     {
-        var parcel = parcels.Find(p => p.Id == id);
-        if (parcel != null)
-        {
-            parcels.Remove(parcel);
-            SaveParcels();
-            Console.WriteLine($"Parcel with ID {id} has been removed.");
-        }
-        else
-        {
-            Console.WriteLine($"Parcel with ID {id} not found.");
-        }
+        fileManager.Remove(id, parcels);
```

Comment:
This logic should not be inside fileManager, it should be here. FileManager should only have two method `Save` and `Read`